test/contracts/common/safe_erc20.js

```diff
@@ -79,52 +80,52 @@ contract('SafeERC20', ([owner, receiver]) => {
         const receipt = await safeERC20Mock.approve(tokenMock.address, receiver, preApprovedAmount - 500)
 
         assertMockResult(receipt, false)
-        assert.equal((await tokenMock.allowance(safeERC20Mock.address, receiver)).valueOf(), preApprovedAmount, 'Allowance of receiver should be the pre-existing value')
+        assertBn(await tokenMock.allowance(safeERC20Mock.address, receiver), preApprovedAmount, 'Allowance of receiver should be the pre-existing value')
       })
 
       it('can correctly transferFrom', async () => {
         // Create approval
-        const approvedAmount = 5000
+        const approvedAmount = bn(5000)
         await tokenMock.approve(safeERC20Mock.address, approvedAmount)
 
         // Transfer to receiver through the mock
         const receipt = await safeERC20Mock.transferFrom(tokenMock.address, owner, receiver, approvedAmount)
         assertMockResult(receipt, true)
-        assert.equal((await tokenMock.balanceOf(owner)).valueOf(), initialBalance - approvedAmount, 'Balance of owner should be correct')
-        assert.equal((await tokenMock.balanceOf(receiver)).valueOf(), approvedAmount, 'Balance of receiver should be correct')
-        assert.equal((await tokenMock.balanceOf(safeERC20Mock.address)).valueOf(), 0, 'Balance of mock should stay the same')
+        assertBn(await tokenMock.balanceOf(owner), initialBalance - approvedAmount, 'Balance of owner should be correct')
```

Comment:
Not sure, but are we able to do math on BNs like this? We don't need to use `sub()`?

Reply:
Yea, it shouldn't work this way, let me fix it anyway